src/centrality.rs

```diff
@@ -165,6 +165,29 @@ pub fn digraph_betweenness_centrality(
     }
 }
 
+/// Compute the degree centrality for nodes in a PyGraph.
+///
+/// Degree centrality assigns an importance score based simply on the number of edges held by each node.
+///
+/// :param PyGraph graph: The input graph
+///
+/// :returns: a read-only dict-like object whose keys are the node indices and values are the
+///      centrality score for each node.
+/// :rtype: CentralityMapping
+#[pyfunction]
+#[pyo3(text_signature = "(graph, /)")]
+pub fn degree_centrality_py(graph: &graph::PyGraph) -> PyResult<CentralityMapping> {
```

Comment:
```suggestion
pub fn graph_degree_centrality(graph: &graph::PyGraph) -> PyResult<CentralityMapping> {
```
Stick to the existing naming convetion